testing/cloudbuild-templates/cloud-run-template.cloudbuild.yaml

```diff
@@ -58,7 +57,7 @@ steps:
   - |
     set -x
     gcloud --quiet container images delete gcr.io/${PROJECT_ID}/${_SERVICE}:${SHORT_SHA}
-    gcloud --quiet run services delete ${_SERVICE}-$(cat _short_id) --region ${_SERVICE_REGION} --platform managed
+    gcloud --quiet run services delete ${_SERVICE}-$BUILD_ID --region ${_SERVICE_REGION} --platform managed
```

Comment:
Out of scope: `--platform managed` unneeded.

Reply:
updated